internal/storage/postgres/flowerbed/flowerbed_test.go

```diff
@@ -313,3 +313,109 @@ func TestTreeClusterRepository_Archived(t *testing.T) {
 		assert.Error(t, err)
 	})
 }
+
+func TestFlowerbedRepository_UnlinkSensorID(t *testing.T) {
+	t.Run("should unlink sensor ID successfully", func(t *testing.T) {
+		// given
+		suite.ResetDB(t)
+		suite.InsertSeed(t, "internal/storage/postgres/seed/test/flowerbed")
+		r := NewFlowerbedRepository(suite.Store, mappers)
+
+		// when
+		err := r.UnlinkSensorID(context.Background(), "1")
+
+		// then
+		assert.NoError(t, err)
```

Comment:
Please check if the sensorID field in the flowerbed is really not linked anymore.